web-ng/src/app/shared/models/user.model.ts

```diff
@@ -19,4 +19,5 @@ export interface User {
   email: string;
   displayName?: string;
   photoURL?: string;
+  uid?: string;
```

Comment:
Rather than add another identifier, you can add a boolean "isAuthenticated" which will be set to `true` when creating real `Users` and `false` in the `ANONYMOUS_USER` constant.